docs/plugins.md

```diff
@@ -0,0 +1,262 @@
+Protractor Plugins
+=================
+
+Plugins extend Protractor's base features by using hooks during test
+execution to gather more data and potentially modify the test output.
+
+The Protractor API and available plugins are *BETA* and may change
+without a major version bump.
+
+The `plugins` folder contains default plugins for Protractor.
+
+##In this document:
+* Using Plugins
```

Comment:
I'm confused.  I think you just write the link as `[Using Plugins](#using-plugins)` and github handles the rest?